resource-managers/kubernetes/integration-tests/pom.xml

```diff
@@ -216,6 +216,35 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <!-- Triggers scalatest plugin in the integration-test phase instead of
+             the test phase, so that test jobs are copied over beforehand.
+             See copy-test-spark-jobs execution of maven-dependency-plugin above. -->
+        <groupId>org.scalatest</groupId>
+        <artifactId>scalatest-maven-plugin</artifactId>
+        <configuration>...</configuration>
+        <executions>
+          <execution>
+            <id>test</id>
+            <goals>
+              <goal>test</goal>
+            </goals>
+            <configuration>
+              <suffixes>(?&lt;!Suite)</suffixes>
```

Comment:
I guess the purpose of this negative pattern is to prevent the `KubernetesSuite` from being tested in the test phase. Better add a comment to explain this?